src/main/java/hudson/plugins/jabber/im/transport/JabberIMConnection.java

```diff
@@ -312,13 +311,25 @@ private boolean createConnection() throws XMPPException, SaslException,
 		
 		String serviceName = desc.getServiceName();
 		final XMPPTCPConnectionConfiguration.Builder cfg = XMPPTCPConnectionConfiguration.builder();
+		
+		if (pi.getProxyType() == ProxyType.NONE) {
+			cfg.setSocketFactory(SocketFactory.getDefault());
```

Comment:
That is what used to happen (in three places below here), but that did not work for me